### ISSUE TITLE
[Global] Missing landmark elements

### ISSUE THREAD

**Issue by svinkle** [CovidShield/portal#12]
## Issue summary

Landmark elements are missing from each page. Landmarks allow screen reader users the ability to navigate around pages quickly and discover content. They also communicate semantic meaning of the page structure.

<img width="814" alt="Screen Shot 2020-05-18 at 12 55 26 PM" src="https://user-images.githubusercontent.com/1392632/82239538-eb9cd980-9906-11ea-80b1-cec07c923644.png">

#### HTML

```html
<div class="top-bar__container">
  <!-- … -->
</div>

<!-- … -->

<div class="page__container">
  <!-- … -->
</div>
```

## Steps to reproduce

1. Go to https://portal-rails.myshopify.io/ with with any supported browser
2. Login with `admin`/`password`
3. With an active screen reader, navigate via Landmark elements only

### Expected

- Inclusion of HTML Landmarks to provide context and navigation.

### Actual

- Landmarks are missing.

## Recommendation

1. Adjust the `.top-bar__container` element to be `header`.
2. Adjust the `.page__container` element to be `main`.

#### HTML

```html
<header class="top-bar__container">
  <!-- … -->
</header>

<!-- … -->

<main class="page__container">
  <!-- … -->
</main>
```

## Specifications

- **Component:** Global
- **WCAG Principle:** Perceivable
- **WCAG SC:** 1.3.1 Info and Relationships
- **Severity:** Medium
- **Effort:** Low
